apps/web/src/hooks/useAssetLogoSource.ts

```diff
@@ -49,7 +49,7 @@ export function getInitialUrl(
   const networkName = chainId ? chainIdToNetworkName(chainId) : 'ethereum'
   const checksummedAddress = isAddress(address)
 
-  if (chainId && isCelo(chainId) && address === nativeOnChain(chainId).wrapped.address) {
+  if (chainId && address === nativeOnChain(chainId).wrapped.address) {
     return celoLogo
```

Comment:
Just to confirm, is this correct? Has `celoLogo` been replaced with ours? or would this never execute?

Reply:
I'll just do it in next PR 😅